packages/web/src/scss/components/UNSTABLE_EmptyState/index.html

```diff
@@ -13,14 +13,14 @@ <h2 class="docs-Heading">Default</h2>
                   stroke-dasharray="4 4"></path>
           </svg>
           <div class="docs-Placeholder__text">
-            <div class="typography-heading-xsmall-text">Placeholder</div>
-            <p class="typography-body-medium-text-regular">Replace me with your own component</p></div>
+            <div class="typography-headline-xsmall-regular">Placeholder</div>
+            <p class="typography-body-medium-regular">Replace me with your own component</p></div>
         </div>
       </div>
       <div class="Stack Stack--hasSpacing UNSTABLE_EmptyState__section"
            style="--stack-spacing: var(--spirit-space-500);">
-        <h2 class="typography-heading-xsmall-text">Headline</h2>
-        <p color="secondary" class="typography-body-medium-text-regular">
+        <h2 class="typography-headline-xsmall-regular">Headline</h2>
```

Comment:
It's headline xsmall bold in Figma:

![image](https://github.com/user-attachments/assets/e60925c9-74b7-4792-95ef-035526ad70f5)